libkbfs/test_common.go

```diff
@@ -458,7 +458,7 @@ func RestartCRForTesting(baseCtx context.Context, config Config,
 	ops := kbfsOps.getOpsNoAdd(folderBranch)
 	ops.cr.Restart(baseCtx)
 	// Start a resolution for anything we've missed.
-	if ops.staged {
+	if ops.bid != NullBranchID {
```

Comment:
can you make this use `isMasterBranch` instead? This must have been an oversight.